plugin/storage/integration/fixtures/queries.json

```diff
@@ -81,8 +81,8 @@
       "ServiceName": "query05-service",
       "OperationName": "",
       "Tags": null,
-      "StartTimeMin": "2017-01-26T15:46:31.639875Z",
-      "StartTimeMax": "2017-01-26T17:46:31.639875Z",
+      "StartTimeMin": "2017-01-26T00:00:31.639875Z",
```

Comment:
I don't even remember what I had for breakfast, can you point me to the code in question?